adafruit_dymoscale.py

```diff
@@ -43,7 +44,12 @@ class ScaleReading:
 class DYMOScale:
     """Interface to a DYMO postal scale."""
 
-    def __init__(self, data_pin, units_pin, timeout=1.0):
+    def __init__(
+        self,
+        data_pin: microcontroller.pin,
+        units_pin: microcontroller.pin,
```

Comment:
It's looks like `units_pin` is confusingly NOT `microcontroller.Pin` but rather `DigitalInOut` (from `digitalio`)!  I'm going to file an issue about this since it's counterintuitive, it may be worth changing the name.